docs/source/using/remote_tracking.md

```diff
@@ -201,6 +201,16 @@ with torch.no_grad():
     print('Test Accuracy: {} %'.format(100 * correct / total))
 ```
 
+### SSL support 
+
+Aim Remote Tracking server can be set up to use SSL certificates to run on secure mode.
+To enable secure mode for server provide `--ssl-keyfile` and `--ssl-certfile` arguments to `aim server `command,
+where `--ssl-keyfile` is the path to the private key file for the certificate and `--ssl-certfile` is the path to the signed certificate.
+
+On client side set `__AIM_CLIENT_SSL_CERTIFICATES_FILE__` environment variable as the path to the PEM encoded root certificates for the secure channel establishment with the server.
```

Comment:
Suggestion:

In order to establish a secure connection with the server, the client has to be configured accordingly.
Please set `__AIM_CLIENT_SSL_CERTIFICATES_FILE__` environment variable to the root directory, where PEM encoded certificates are located, e.g. 

```shell
export __AIM_CLIENT_SSL_CERTIFICATES_FILE__=/path/of/the/certs
```